boot/zephyr/Kconfig.serial_recovery

```diff
@@ -65,8 +65,8 @@ config MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD
 
 config BOOT_SERIAL_UNALIGNED_BUFFER_SIZE
 	int "Stack buffer for unaligned memory writes"
-	default 64
-	range 0 128
+	default 512
+	range 0 1024
```

Comment:
the default should be changed for your boards/socs, not every board. There is no reason every board with serial recovery should suddenly have an extra 448 bytes of RAM taken up for literally nothing

Reply:
Hi @nordicjm 
The goal to have the same user experience as for Zephyr MCUMgr that works for all cases out of the box.
Having the higher value bring only benefits.
It works for all boards, faster, and without any resource penalty as the MCUBoot stack is too high - set to 10KB (used <20%).

Reply:
Sure, but you bring surprise to people who have tuned down stack and build of MCUboot, after this is brought in, topples into stack fault.
So now they have to go to their config files and bring that down.
I understand what you are trying to fix but you are not fixing that for everyone, quite an opposite.
The right approach is to have tuning done for device/soc/board.

Most people are not aware of the config or just briefly know of existence, because they have never had to deal with it as default worked.

Reply:
> Sure, but you bring surprise to people who have tuned down stack and build of MCUboot, after this is brought in, topples into stack fault. So now they have to go to their config files and bring that down. I understand what you are trying to fix but you are not fixing that for everyone, quite an opposite. The right approach is to have tuning done for device/soc/board.
> 
> Most people are not aware of the config or just briefly know of existence, because they have never had to deal with it as default worked.

Updated. 
Default value increased only for LPC55 and MCXN, for all other platforms unchanged.